chalicelib_fourfront/checks/wrangler_checks.py

```diff
@@ -1702,24 +1703,41 @@ def states_files_without_higlass_defaults(connection, **kwargs):
     # add random wait
     wait = round(random.uniform(0.1, random_wait), 1)
     time.sleep(wait)
+    valid_tags = ['SPIN_states_v1']  # only 1 at the moment but make the current tag the first in the list
     query = '/search/?file_type=chromatin states&type=File'
     res = ff_utils.search_metadata(query, key=connection.ff_keys)
+    updates = {}
     for a_res in res:
-        if not a_res.get('higlass_defaults'):
-            if not a_res.get('tags'):
-                check.full_output['problematic_files'][a_res['accession']] = 'missing state tag'
-            else:
-                check.full_output['to_add'][a_res['accession']] = a_res["tags"]
-
-    if check.full_output['to_add']:
+        uid = a_res.get('uuid')
+        acc = a_res.get('accession')
+        has_higlass_defaults = False
+        has_tag = False
+        import pdb; pdb.set_trace()
```

Comment:
@aschroed First of all, I apologize for commenting on this PR even though I wasn't assigned to it, but this line caught my attention. Should it stay like this, or was it overlooked? That's why I wanted to mention it.

Reply:
Thanks @utku-ozturk - overlooked and you should always feel free to comment - especially if you find something like this.